instrumentation/opentelemetry-instrumentation-tortoiseorm/src/opentelemetry/instrumentation/tortoiseorm/__init__.py

```diff
@@ -0,0 +1,214 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Instrument `tortoise-orm`_ to report SQL queries.
+Usage
+-----
+.. code:: python
+    from opentelemetry.instrumentation.tortoiseorm import TortoiseORMInstrumentor
+    from tortoise.contrib.fastapi import register_tortoise
+
+    register_tortoise(
+        app,
+        db_url=settings.db_url,
+        modules={"models": ["example_app.db_models"]},
+        generate_schemas=True,
+        add_exception_handlers=True,
+    )
+
+    TortoiseORMInstrumentor().instrument(tracer_provider=tracer)
```

Comment:
Generally instumentors are activated before a package is imported. Any reason to recommend doing it after here?

Reply:
no reason, I'll swap it around.